monitoring/main/main.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
is this an alternative to unit testing ?

Reply:
Not quite, I use it to see the prints and make sure everything is ok instead of running auto tests.